cpp/include/games/nim/Game.hpp

```diff
@@ -165,4 +123,7 @@ struct hash<nim::Game::State> {
 };
 }  // namespace std
 
-static_assert(core::concepts::Game<nim::Game>);
\ No newline at end of file
+static_assert(core::concepts::Game<nim::Game>);
+
+#include <inline/games/nim/Game.inl>
+
```

Comment:
Please make a new `stochastic_nim::Game`, distinct from the original `nim::Game`. Leave the original nim game along with unit-tests in-tact.